backend/program/scrapers/torrentio.py

```diff
@@ -20,9 +21,7 @@ def __init__(self, media_items: MediaItemContainer):
         self.class_settings = settings_manager.get(self.settings)
         self.last_scrape = 0
         self.filters = self.class_settings["filter"]
-        self.minute_limiter = RateLimiter(
-            max_calls=60, period=60, raise_on_limit=True
-        )
+        self.minute_limiter = RateLimiter(max_calls=60, period=60, raise_on_limit=True)

```

Comment:
Looks like your fixing black's formatting

Reply:
Then black formatted its formatting itself, I just ran it again 🗡️